fili-core/src/main/java/com/yahoo/bard/webservice/web/endpoints/DataServlet.java

```diff
@@ -85,6 +74,15 @@
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.PathSegment;
 import javax.ws.rs.core.UriInfo;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+
+import static com.yahoo.bard.webservice.web.handlers.workflow.DruidWorkflow.REQUEST_WORKFLOW_TIMER;
+import static javax.ws.rs.core.Response.Status.BAD_REQUEST;
+import static javax.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR;
+import static javax.ws.rs.core.Response.Status.REQUEST_TIMEOUT;
```

Comment:
Looks like your import configuration doesn't agree with ours. The imports should be static, then `com.yahoo` then `com` then `org` then other, then `java`, then `javax`.